<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindCorrelationWidget</class>
 <widget class="QWidget" name="FindCorrelationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>428</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Find Correlations</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="3">
    <widget class="QCheckBox" name="directly_proportional_checkBox">
     <property name="text">
      <string>Directly proportional</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QCheckBox" name="inversely_proportional_checkBox">
     <property name="text">
      <string> inversely proportional</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2" colspan="5">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Angle Error (rad)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2" rowspan="2">
      <widget class="QDoubleSpinBox" name="angle_error">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.001000000000000</double>
       </property>
       <property name="maximum">
        <double>99.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>% Distance Error</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="distance_error">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.001000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QRadioButton" name="allcorrelations">
       <property name="text">
        <string>Find all correlations</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="correlationsto">
       <property name="text">
        <string>Find correlated to</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="variables">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="correlations_from_list">
       <property name="text">
        <string>Find correlated to variable list</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QComboBox" name="varlist">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>147</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QPushButton" name="findButton">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>155</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" colspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="4">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="4" column="1">
      <widget class="QPushButton" name="highlightButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>188</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Highlight Selected</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>72</width>
         <height>38</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="invertselectionButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Invert Selection</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="6">
      <widget class="QListView" name="listView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="selectallButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Select All</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="unselectallButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Unselect All</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="plotButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>188</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Plot Selected</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1" colspan="4">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>168</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="closeButton">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>inversely_proportional_checkBox</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>findButton</zorder>
  <zorder>horizontalSpacer_4</zorder>
  <zorder>horizontalSpacer_5</zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder>directly_proportional_checkBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
